### DISCUSSION TITLE
Event backup with JSON archive - examples on how to configure App Settings

### DISCUSSION THREAD

**Discussion by rodneytamblyn** [datalust/seq-tickets#2185] [Help]
Further to: https://docs.datalust.co/docs/event-backup-with-json-archive

Can someone provide more detail on how to configure the App Settings for JSON Archive.  For example, by default are all events logged?  Can I restrict events to a certain signal (e.g. info level events)?  Can someone provide a few examples so I can get more of an idea of what is possible?

Thanks in advance...

Rod

```
{
  "fileSet": "/data/MYARCHIVEPATH/infoevents_*.json",
  "readableFilenames": "True",
  "chunkSize": null
}
```

**Comment by liammclennan** [comment]
Hi Rod,

By default events are not streamed to apps. Apps only receive events that are sent to them. For JSON archive you will probably want to enable 'Stream incoming events' - then you choose a set of signals to reduce the events that are written to the file.
